lib/errors_savior/saviors/base_savior.rb

```diff
@@ -0,0 +1,47 @@
+module ErrorsSavior
+  module Saviors
+    class BaseSavior
+      # self.abstract_class = true
```

Comment:
Is there any reason to have this comment? :thinking:

Reply:
[`.abstract_class`](https://api.rubyonrails.org/classes/ActiveRecord/Inheritance/ClassMethods.html#method-i-abstract_class-3F)  it's an ActiveRecord method, but it's an abstract class :joy: we don't need to initialize this